src/libs/ajax.js

```diff
@@ -723,6 +723,14 @@ const Workspaces = signal => ({
         return res.json()
       },
 
+      lock: async () => {
+        return await fetchRawls(`${root}/lock`, _.merge(authOpts(), { signal, method: 'PUT' }))
```

Comment:
These return the `fetch` response objects, whereas other methods here return the JSON decoded response body. It doesn't look like anything currently uses the return value, but anyone attempting to in the future might be surprised.

Reply:
You're right.

Alternatively, you could just await the fetchRawls request and not return anything.